src/app/clusters/level-control/level-control.cpp

```diff
@@ -408,23 +417,22 @@ static bool shouldExecuteIfOff(EndpointId endpoint, CommandId commandId, uint8_t
         // ---------- The following order is important in decision making -------
         // -----------more readable ----------
         //
-        if (optionsMask == 0xFF && optionsOverride == 0xFF)
```

Comment:
According to the comment [above ](https://github.com/project-chip/connectedhomeip/pull/23656/files#diff-0b846b0cf4124840f4d87631d664fed23f7529ce064d963b5021c003176516a9R404), the old check was actually wrong, as it states both fields SHALL be present or both fields SHALL be omitted. Thus changed && to ||